src/mongocxx/options/find.hpp

```diff
@@ -26,13 +26,14 @@ namespace mongocxx {
 MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
+enum class cursor_type : std::uint8_t { k_non_tailable, k_tailable, k_tailable_await };
```

Comment:
The issue is when trying to get a value (ex: `k_tailable` value). The compiler can't resolve `mongocxx::options::find::cursor_type::k_tailable`.